data_service/src/match_processing/match_parser.rs

```diff
@@ -0,0 +1,98 @@
+use riven::models::match_v5::Match;
+
+use crate::types::PuuidToChampionMapping;
+
+pub async fn parse_match(match_data: Match) -> Vec<PuuidToChampionMapping> {
+    // Get match data.
+
+    let to_return: Vec<PuuidToChampionMapping> = match_data
+        .info
+        .participants
+        .iter()
+        .map(|participant| PuuidToChampionMapping {
+            puuid: participant.puuid.clone(),
+            champion_name: participant.champion_name.clone(),
+        })
+        .collect();
+
+    // println!("Printing mappings");
```

Comment:
@saikhurana98 probably you should remove these comments

Reply:
resolved, please check